package dev.cmanrg.movie.feature.home.data.remote.dto

import com.google.gson.annotations.SerializedName

data class MovieResponseDto(
    val page: Int,
    @SerializedName("total_pages") val totalPages: Int,
    @SerializedName("total_results") val totalResults: Int,
    val results: List<MovieDto>
)
